tap_square/client.py

```diff
@@ -162,23 +162,20 @@ def get_bank_accounts(self):
             body,
             'bank_accounts')
 
-    def get_customers(self, start_time, bookmarked_cursor):
+    def get_customers(self, start_time, end_time, cursor):
         body = {
             "query": {
                 "filter": {
                     "updated_at": {
-                        "start_at": start_time
+                        "start_at": start_time, # Inclusive on start_at
+                        'end_at': end_time      # Exclusive on end_at
                     }
                 },
-                "sort": {
-                    "sort_field": "UPDATED_AT",
-                    "sort_order": "ASC"
-                }
             }
         }
 
-        if bookmarked_cursor:
-            body['cursor'] = bookmarked_cursor
+        if cursor is not None:
```

Comment:
This is so nitpicky I almost want you to ignore this comment, but even though I know this is more explicit, it's usually more pythonic to just do `if cursor:` unless there's a reason to suspect other non-truthy values could be provided, like false or blank... 

To be more specific after writing the above I looked up where the style suggestion comes from, it's google / pep8:
https://google.github.io/styleguide/pyguide.html?showone=True/False_evaluations#2144-decision

>Use the “implicit” false if possible, e.g., if foo: rather than if foo != []:. There are a few caveats that you should keep in mind though:

>* Always use if foo is None: (or is not None) to check for a None value. E.g., when testing whether a variable or argument that defaults to None was set to some other value. The other value might be a value that’s false in a boolean context!

So the question then becomes is the default above None? You're not writing in a default, but maybe everyone agrees the default behavior should only apply for None? Again, I think I'm overthinking this... so feel free to decide whatever you prefer.

Reply:
On line 85 in sync.py we are explicitly setting the cursor to `None` and here we are checking if it is that value.